Elevel/Elevel.Api/Controllers/BaseApiController.cs

```diff
@@ -13,7 +13,7 @@ namespace Elevel.Api.Controllers
     [ApiController]
     public abstract class BaseApiController : ControllerBase
     {
-        private IMediator _mediator;
+        internal IMediator _mediator;
```

Comment:
you should not change this
you can use Mediator on row below

Reply:
Do you mean i should add extra IMediator field with internal modificator?



> you should not change this
> you can use Mediator on row below